src/quantity.rs

```diff
@@ -113,6 +113,25 @@ macro_rules! quantity {
             }
         }
     };
+    (
+        $(#[$quantity_attr:meta])* quantity: $quantity:ident; $description:expr;
+        alias_of: $alias_of:ident;
```

Comment:
Why add the alias? In my first brief look at the code it looks like it allows the alias's units to be used?

Quantities with the same dimension and kind are already type aliases although I'm they can't share units currently.

```rust
type Energy = Quantity<m^2 * kg * s^-2>;
type Torque = Quantity<m^2 * kg * s^-2>;
```

Reply:
It's so the `Conversion` can extend the `Unit` of both the original type and the alias.